proto/teleport/lib/prehog/v1alpha/connect.proto

```diff
@@ -18,7 +18,7 @@ package prehog.v1alpha;
 
```

Comment:
Let's move the files in this folder to `proto/prehog/v1alpha`, so it correctly matches the package name.

Reply:
> Let's move the files in this folder to `proto/prehog/v1alpha`, so it correctly matches the package name.

Wouldn't it be better to leave it as it is and then ignore `PACKAGE_DIRECTORY_MATCH` for that directory? This way if someone wants to add another directory to `proto/teleport/lib`, they'll clearly see what convention other directories follow. prehog's proto package name will be an outlier, explained by the comment in buf.yaml which explains why the rule is ignored for that package.

Reply:
I see your point but I'd rather not break package naming conventions, as it has knock-on effects on proto imports and generated package names. Hopefully we spot new protos before they get placed weirdly. (Also, a long TODO of mine is to do a writeup on this. I should get to it.)